appveyor.yml

```diff
@@ -57,6 +57,8 @@ build_script:
 
 environment:
   NOT_CRAN: true
+  # See SPARK-29378.
+  ARROW_PRE_0_15_IPC_FORMAT: 1
```

Comment:
Does setting this here carry it through to the worker?

Reply:
I removed it in the latest commit :-).